Eastern-Research-Group/expert-query
# Feature/96 add rest of mvs to etl

## Related Issues:
* [EQ-96](https://jira.epa.gov/browse/EQ-96)
* [EQ-97](https://jira.epa.gov/browse/EQ-97)
* [EQ-98](https://jira.epa.gov/browse/EQ-98)
* [EQ-102](https://jira.epa.gov/browse/EQ-102)
* [EQ-62](https://jira.epa.gov/browse/EQ-62)
* [EQ-63](https://jira.epa.gov/browse/EQ-63)
* [EQ-64](https://jira.epa.gov/browse/EQ-64)
* [EQ-68](https://jira.epa.gov/browse/EQ-68)

## Main Changes:
* Added the following profiles to the etl process and data API: actions, assessments, catchment correspondence and tmdl
  * The assessments and catchment correspondence MVs are still unreliable, so I commented those out and put in a TODO comment
* Removed the `gisAssessments` table and API endpoint

## Steps To Test:
1. In the `etl/app/server/profiles/index.js` file, uncomment lines 4 and 7
2. Run `npm run etl_database`
    * You can hit `CTRL + C` after all of the tables are created, because the `assessments` and `catchment correspondence` tables will fail to get data. These MVs will take a long time before they fail
3. Verify all of the tables are created
4. In the `etl/app/server/profiles/index.js` file, comment lines 4 and 7
5. Run `npm run etl_database`, this time let it run all the way through
6. Verify all of the MVs except for `assessments` and `catchment correspondence` are created and loaded with data
7. Navigate to the following URLs to verify data API works:
    * http://localhost:9090/attains/data/actions?id=1&id=2 - This should return data
    * http://localhost:9090/attains/data/assessments?id=1&id=2 - This should return a `relation ... does not exist` which is expected for now
    * http://localhost:9090/attains/data/catchmentCorrespondence?id=1&id=2 - This should return a `relation ... does not exist` which is expected for now
    * http://localhost:9090/attains/data/tmdl?id=1&id=2 - This should return data

## TODO:
- [ ] Add the `assessments` and `catchment correspondence` profiles back in when their MVs are stable
